app/views/qsets/_question_list_filter.html.erb

```diff
@@ -0,0 +1,83 @@
+<div class="row question_list">
+
+  <div class="panel-group no-questions hidden panel panel-default" role="dialog">
```

Comment:
Why `role="dialog"`? That's usually for modals or overlays, not for something like this, I think.